cluster-controller/src/main/java/io/strimzi/controller/cluster/ClusterController.java

```diff
@@ -232,9 +233,9 @@ private void startHealthServer() {
                 .requestHandler(request -> {
 
                     if (request.path().equals("/healthy")) {
-                        request.response().setStatusCode(HttpResponseStatus.OK.code()).end();
+                        request.response().setStatusCode(200).end();
```

Comment:
While you're right that it doesn't remove netty as a transitive dependency, using it requires us to add it as a direct dependency. That, in turn, requires we specify a version, which opens us up to the versions (direct and transitive) diverging in the future, which is a problem because these things end up on a flat classpath and so our old version might override the newer version required by vertx and cause linking errors at runtime.

Now balance that against what we're actually using from netty here: A single constant whose value is, and always will be, 200. That 200, in context, has a pretty well-know meaning, so I don't think there's even *much* value in using a constant (even one we define ourselve): The meaning is clear and it's hard to mistype the literal `200`.

I guess we could add a constant to `ClusterController` if you really felt using a constant was justified, but adding a dependency just to use a single constant whose value we know and know won't ever change seems like the wrong way to solve this problem.

Reply:
Ok. In that case, you could probably remove `<netty.version>4.1.15.Final</netty.version>` from `pom.xml`, or?

> I guess we could add a constant to ClusterController if you really felt using a constant was justified, but adding a dependency just to use a single constant whose value we know and know won't ever change seems like the wrong way to solve this problem.

TBH, IIRC I used to use the numbers as well ... I have no strong feeling about this. But @ppatierno suggested to use the Netty class ... so lets see what he things about it.

Reply:
In general, I don't like having literals in the code maybe because as an "old" embedded developer having #define and constants is much better for readability and for code manageability and they have no cost at binary level. I didn't imagine to have such a problem on dependencies :-) so let's stay with the "bad" `200` literal. :P